narwhals/expr.py

```diff
@@ -706,6 +706,52 @@ def std(self, *, ddof: int = 1) -> Self:
         """
         return self.__class__(lambda plx: self._to_compliant_expr(plx).std(ddof=ddof))
 
+    def var(self, *, ddof: int = 1) -> Self:
+        """Get variance.
+
+        Returns:
+            A new expression.
```

Comment:
Could you add `Argument` section:

```suggestion

        Arguments:
            ddof: "Delta Degrees of Freedom": the divisor used in the calculation is N - ddof,
                where N represents the number of elements. By default ddof is 1.
        
        Returns:
            A new expression.
```